modelRaceFromQuali.py

```diff
@@ -0,0 +1,44 @@
+import random
+
+from python import *
+
+import pickle
+
+class F1RaceFromQualiModel:
+    '''Predicts race results to be equal to qualifying results'''
+
+    def __init__(self, raceSeasonsData, raceResultsData, qualiResultsData):
+        self.seasonsData = seasonsData
+        self.raceResultsData = raceResultsData
+        self.qualiResultsData = qualiResultsData
+
+    def constructPredictions(self):
+        predictions = []
+
+        for year, season in self.seasonsData.items():  # Read every season:
+            racesAsList = list(season.races.items())
+            racesAsList.sort(key=lambda x: x[1].round)
+            for raceId, data in racesAsList:
+                # A single race
+                if raceId in self.raceResultsData and raceId in self.qualiResultsData:
+                    qualiResults = self.qualiResultsData[raceId]
+                    predictedRaceResults = []
+                    for i in range(len(qualiResults)):
+                    	predictedRaceResults.append(qualiResults[i])
```

Comment:
This appends the whole qualiResult tuple, e.g. (832, 1, 92.84700000000001) when we only want the driver id, which seems to be the first number in the tuple.